datasets/wmt20_mlqe_task1/dataset_infos.json

```diff
@@ -0,0 +1 @@
ormation for the task.\n\nTask 1 uses Wikipedia data for 6 language pairs that includes high-resource\nEnglish--German (En-De) and English--Chinese (En-Zh), medium-resource\nRomanian--English (Ro-En) and Estonian--English (Et-En), and low-resource\nSinhalese--English (Si-En) and Nepalese--English (Ne-En), as well as a\ndataset with a combination of Wikipedia articles and Reddit articles\nfor Russian-English (En-Ru). The datasets were collected by translating\nsentences sampled from source language articles using state-of-the-art NMT\nmodels built using the fairseq toolkit and annotated with Direct Assessment (DA)\nscores by professional translators. Each sentence was annotated following the\nFLORES setup, which presents a form of DA, where at least three professional\ntranslators rate each sentence from 0-100 according to the perceived translation\nquality. DA scores are standardised using the z-score by rater. Participating systems\nare required to score sentences according to z-standardised DA scores.\n", "citation": "\nNot available.\n", "homepage": "http://www.statmt.org/wmt20/quality-estimation-task.html", "license": "Unknown", "features": {"segid": {"dtype": "int32", "id": null, "_type": "Value"}, "translation": {"languages": ["ru", "en"], "id": null, "_type": "Translation"}, "scores": {"feature": {"dtype": "float32", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "mean": {"dtype": "float32", "id": null, "_type": "Value"}, "z_scores": {"feature": {"dtype": "float32", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "z_mean": {"dtype": "float32", "id": null, "_type": "Value"}, "model_score": {"dtype": "float32", "id": null, "_type": "Value"}, "doc_id": {"dtype": "string", "id": null, "_type": "Value"}, "nmt_output": {"dtype": "string", "id": null, "_type": "Value"}, "word_probas": {"feature": {"dtype": "float32", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}}, "post_processed": null, "supervised_keys": null, "builder_name": "wmt_mlqe_task1", "config_name": "ru-en", "version": {"version_str": "1.1.0", "description": null, "major": 1, "minor": 1, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 3520776, "num_examples": 7000, "dataset_name": "wmt_mlqe_task1"}, "test": {"name": "test", "num_bytes": 478787, "num_examples": 1000, "dataset_name": "wmt_mlqe_task1"}, "validation": {"name": "validation", "num_bytes": 499601, "num_examples": 1000, "dataset_name": "wmt_mlqe_task1"}}, "download_checksums": {"https://www.quest.dcs.shef.ac.uk/wmt20_files_qe/ru-en.tar.gz": {"num_bytes": 1897111, "checksum": "a1b762e4c21195b3c8e65e65ab5fb75527d4f8d3ce54e34de18ef773d22f675f"}, "https://www.quest.dcs.shef.ac.uk/wmt20_files_qe/ru-en_test.tar.gz": {"num_bytes": 226573, "checksum": "9c1d35a2b3f5d807f5fa54ae9114ba8384dcd2929452435b54306efe1d501956"}}, "download_size": 2123684, "post_processing_size": null, "dataset_size": 4499164, "size_in_bytes": 6622848}}
```

Comment:
Can you regenerate this file after the class name change please ?

Also it looks like it has metadata about previous config names that you may have generated in the past.
So please delete this file and clear your cache before regenerating it.
For example the config `en-de-task1` doesn't exist anymore